test/test_ops.py

```diff
@@ -1021,7 +1021,7 @@ def test_convert_boxes_to_roi_format(self, box_sequence):
             assert_equal(ref_tensor, ops._utils.convert_boxes_to_roi_format(box_sequence))
 
 
-class TestBox:
+class TestBoxConvert:
```

Comment:
We have `TestBoxConversion` just above which is very similar. Should we merge them? Or find less conflicting names?